build/MUX-Release.yml

```diff
@@ -135,3 +136,25 @@ jobs:
       Release_arm:
         buildPlatform: 'arm'
         buildConfiguration: 'Release'
+
+- job: WACKTests
+  dependsOn: BuildNugetPkgTests
+  pool:
+    name: Package ES Lab E
+  strategy:
+    maxParallel: 10
+    matrix:
+      Release_x86:
+        buildPlatform: 'x86'
+        buildConfiguration: 'Release'
+      Release_x64:
+        buildPlatform: 'x64'
+        buildConfiguration: 'Release'
+      Release_Arm:
+        buildPlatform: 'arm'
+        buildConfiguration: 'Release'
+  steps:
+  - task: PkgESWACKTests@10
+    inputs:
+      AppxFolder: '$(artifactDownloadPath)\NugetPkgTestsDrop\$(BuildConfiguration)\$(BuildPlatform)\NugetPackageTestApp\AppPackages\NugetPackageTestApp_Test'
```

Comment:
What about the "framework package" version of the nuget package tests? Does this cover both?

Reply:
We are not building framework package tests in release pipeline now. Should I add it?